roles/setup/vars/RedHat.yml

```diff
@@ -1,3 +1,4 @@
 __receptor_packages:
   - receptor
+  - receptorctl
```

Comment:
this only cover the RHEL scenario where does other distro get receptorctl? 

if its in the install bundle should we try to bring it in this collection?

Reply:
yeah if no on RHEL, then it will install via pypi

https://github.com/ansible/receptor-collection/pull/39/files#diff-72f8b038e4682c96f61925416b11504c86329d965daa54b9f54626f94d64b15eR30